lib/order.rb

```diff
@@ -0,0 +1,67 @@
+require './lib/menu.rb'
+require './lib/confirm_order.rb'
+
+class Order
+  attr_reader :order_list, :total
+  
+  def initialize
+    @menu = Menu.new
+    @order_list = []
+    @total = 0
+  end
+  
+  def print_menu
+    puts @menu.print_menu
+  end
+
+  def select_dish
```

Comment:
I would put these as private methods, as when I was running your code, I started trying to use these methods and didn't understand why they didn't add a dish to the `order_list` but I realise they get called when you call `order_item`. So if your method is not called outside of your class and is only used by another method, and you don't want Joe Public messing about with them, you can make them private methods.